### PR TITLE
Feature popup

### PR DESCRIPTION
**Pull request by MeineZ**
This pull request adds a class to spawn popups within Maya.
It also improved the initial popup on startup (cleaned up a lot of hardcoded code).
It also adds a 'one-time' popup on a switch to the hybrid pipeline (Real-time Final Render Preview). This popup only opens if dxr is not supported. This warns the user that it may crash Maya. The user will have to open the pipeline again if he's really sure about it.

## Types of changes
- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Checklist:
- [x] My code follows the code style of this project.
- [x] My change requires a change to the documentation.
- [x] I have updated the documentation accordingly.

### CODE REVIEW

File: src/miscellaneous/maya_popup.cpp [hunk 1]
```diff
@@ -0,0 +1,108 @@
+#include "maya_popup.hpp"
+
+// Maya includesd
```
Comment (tntmeijs) [inline src/miscellaneous/maya_popup.cpp]: Typo

File: src/miscellaneous/maya_popup.hpp [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+
+		//! Spawn a popup with text loaded from a file
+		/*!
+			\param path The path to the location of the file
+			\param options [Optional] The options for this popup
+			\return A boolean that defines if the file could be loaded or not. `true` is it was loaded correctly.
+		*/
+		static bool SpawnFromFile(const char * path, const Options& options = Options()) noexcept;
+
+	private:
```
Comment (tntmeijs) [inline src/miscellaneous/maya_popup.hpp]: Unused private keyword

File: src/plugin/renderer/render_pipeline_select_command.cpp [hunk 3]
```diff
@@ -47,7 +49,21 @@ MStatus wmr::RenderPipelineSelectCommand::doIt(const MArgList& args)
...
+			first_time_hybrid = false;
+
+			// Open warning popup
+			MayaPopup::Options options;
+			options.window_name = "hybrid_open_popup";
+			options.width = 500;
+
+			MayaPopup::SpawnFromFile("resources/hybrid_switch.txt", options);
+		}
+		else {
```
Comment (tntmeijs) [inline src/plugin/renderer/render_pipeline_select_command.cpp]: Style guide required '{' and '}' to be on a newline.

**Review by tntmeijs — DISMISSED** [review]
Left you some comments.

**Review by sainsay — APPROVED** [review]

**Review by sainsay — APPROVED** [review]

**Pull request merged**